docs/DeveloperGuide.md

```diff
@@ -244,6 +255,10 @@ instead opting to defer the creation of the new `Task` to the `Model` itself. Th
 code to parse in all the fields of `Task` and leaving student as a list of Strings, which we then use to search through
 the `Model` to find the students that are assigned to the task.
 
+Show below is an activity diagram of how a task is edited.
+
+![Task Edit Activity Diagram](images/TaskEditActivityDiagram.png)
```

Comment:
There is no `TaskEditActivityDiagram.png`. You may have forgotten to add the file.